src/main/java/com/example/springboot/service/impl/TaskHistoryServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package com.example.springboot.service.impl;
+
+import com.example.springboot.dto.TaskHistoryResponseDto;
+import com.example.springboot.exception.NoSuchItemException;
+import com.example.springboot.model.Todo;
+import com.example.springboot.repository.TaskHistoryRepository;
+import com.example.springboot.repository.TodoRepository;
+import com.example.springboot.service.TaskHistoryService;
+import lombok.RequiredArgsConstructor;
+import com.example.springboot.mapper.TaskHistoryMapper;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class TaskHistoryServiceImpl implements TaskHistoryService {
+
+  final private TaskHistoryMapper taskHistoryMapper;
+  final private TaskHistoryRepository taskHistoryRepository;
+  final private TodoRepository todoRepository;
```

Comment:
Here is a correct order:
```suggestion
  private final TaskHistoryMapper taskHistoryMapper;
  private final TaskHistoryRepository taskHistoryRepository;
  private final TodoRepository todoRepository;
```